kaitai/kaitaistream.cpp

```diff
@@ -635,9 +635,10 @@ uint8_t kaitai::kstream::byte_array_max(const std::string val) {
 #include <iconv.h>
 #include <cerrno>
 #include <stdexcept>
+#include "kaitaistream.h"
```

Comment:
Umm..., what? This should have been obviously already included at the very beginning of `kaitai/kaitaistream.cpp`:

https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/blob/2939b29cc38962c931d29087bf0a6872aa767bd6/kaitai/kaitaistream.cpp#L1

So I see no point in doing it for the second time (`kaitaistream.h` has header guards, so it won't have any effect, but... why).

Reply:
Oops, sorry, looks like some "smart" IDE tried to help and added it here, and I've overlooked it. Will fix.